src/Invoice/Inv/Trait/Calendar.php

```
@@ -0,0 +1,176 @@
+    {
+        if ($year !== '' && $month !== '') {
+            $candidate = \DateTimeImmutable::createFromFormat(
+                'Y-n-j',
+                $year . '-' . $month . '-1'
+            );
+            if ($candidate instanceof \DateTimeImmutable) {
+                return $this->calendarModify($candidate, self::FIRST_DAY_OF_MONTH)
+                    ->setTime(0, 0);
+            }
+        }
+        return new \DateTimeImmutable(self::FIRST_DAY_OF_MONTH);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '93', column '20'). (undefined)

(MissingImport)